curriculum/challenges/english/10-coding-interview-prep/rosetta-code/execute-a-markov-algorithm.md

```diff
@@ -42,11 +42,15 @@ a never used -> .terminating rule
 
 Sample text of:
 
-`I bought a B of As from T S.`
+```html
+I bought a B of As from T S.
+```
```

Comment:
@RandellDawson what are your thoughts on this file?

These aren't HTML, but technically JavaScript - though they aren't formatted as strings so would likely throw an error. Would it be better to format them as strings and use `js` as the language, or remove the codeblocks and use `pre` tags (as we do in the ruleset just above this). 

I'm leaning more toward the `pre` tags myself.

Reply:
Or we can make them inline code blocks of the previous sentence like we did in the other PR.

Reply:
So, Was that a consensus on this?  I could make the changes and move forward maybe implementing the last portion of it as well

Reply:
@LaurentLabine The consensus is to make this particular code block an inline code block a part of a sentence.  See my suggestion for the first **Ruleset** that could be applied to the other 4 rulesets.

**Change:**

````md
Sample text of:

```html
I bought a B of As from T S.
```

Should generate the output:

```html
I bought a bag of apples from my brother.
```

````

**to the following:**

````md
Sample text of `I bought a B of As from T S.` should generate the output `I bought a bag of apples from my brother.`
````